wayback-webapp/src/main/webapp/WEB-INF/ComplexAccessPoint.xml

```diff
@@ -161,7 +166,11 @@
         </property>
       </bean>
     </property>
-
+    <property name="trustedProxies">
+      <list>
+        <value>140.147.210.0/24</value>
+      </list>
+    </property>
```

Comment:
Makes sense to me! I made the changes here: https://github.com/chasehd/openwayback/commit/a829598

Let me know if there's anything else.